TestProjects/UniversalGraphicsTest/Packages/manifest.json

```diff
@@ -4,6 +4,7 @@
         "com.unity.2d.sprite": "1.0.0",
         "com.unity.2d.tilemap": "1.0.0",
         "com.unity.ext.nunit": "1.0.0",
+		"com.unity.ide.rider": "2.0.7",
```

Comment:
Yes - otherwise Unity doesn't talk to Rider correctly, it has an empty project and you can't use it without manually adding this package. The VSCode and Visual Studio packages are already included for the same reason

Reply:
FYI: We recently removed this dependency when the Rider package had an error.

Reply:
Okay - it seems fine now, is it good to leave it in?

Reply:
It would remove the annoyance of always having to add it (I use Rider as well) but it does add the danger of our tests failing should it break again.

I personally wouldn't mind having it included again and then removed if we experience that again but I'll let @phi-lira decide.